features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/operationComponents/asyncapi/Runtime.jsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+
+/**
+ *
+ * Renders the callback URL for WebSub and URI mapping for WebSocket.
+ * @export
+ * @param {*} props
+ * @returns
+ */
+export default function Runtime(props) {
+    const {
+        operation, operationsDispatcher, target, verb, api,
+    } = props;
+    operation.runtime = operation.runtime || { };
+
+    const buildCallbackURL = () => {
+        return `https://{GATEWAY_HOST}:9021/${api.context.toLowerCase()}/${api.version}/`
+            + `webhooks_events_receiver_resource?topic=${target.toLowerCase()}`;
```

Comment:
Shall we update this as mentioned in https://github.com/wso2/product-apim/issues/10188 ?

Reply:
Fixed